### PR TITLE
Revert "Revert "Revert "Revert "Translating some of the last LP stories""""

### PR DESCRIPTION
**Pull request by hannahbergam**
A third attempt! Why I think this will work:

I split up the shared use of fake section data between the unit tests and storybook. Some of the section data is purposefully incorrect (a mislabeled login type) to test that those incorrectly formatted sections will not render. I noted as much in a test comment. This was making it difficult to use the section data for storybook while avoiding console errors. 

By adding another array of *correctly formatted* sections, I was able to get all the storybook examples rendering without the prop type errors (which was not blocking a build, but was failing Test).

### CODE REVIEW

File: apps/test/unit/templates/studioHomepages/homepagesTestData.js [hunk 1]
```diff
@@ -167,13 +167,48 @@ export const joinedSections = [
     linkToAssigned: 'https://studio.code.org/s/course4',
     numberOfStudents: 4,
     linkToStudents: manageStudentsUrl,
-    loginType: 'clever',
+    login_type: 'clever',
```
Comment (bethanyaconnor) [inline apps/test/unit/templates/studioHomepages/homepagesTestData.js:170]: why change this back to snake case?

**Review by bethanyaconnor — APPROVED** [review]
🤞

File: apps/test/unit/templates/studioHomepages/homepagesTestData.js (hunk 1 above)
Comment (hannahbergam) [reply apps/test/unit/templates/studioHomepages/homepagesTestData.js:170]: I think it was intentional before- these ones are not supposed to render in the unit tests that use them (see screenshot below of the test data codes). I can add a comment in the code indicating as such. 
Screenshot 2023-06-26 at 9 00 04 AM

Comment (bethanyaconnor) [reply apps/test/unit/templates/studioHomepages/homepagesTestData.js:170]: no worries about a comment! just sad that we've mixed the snake case in with the camel case 🙃

Comment (hannahbergam) [reply apps/test/unit/templates/studioHomepages/homepagesTestData.js:170]: Completely agree... I instinctively changed it to match and it failed those unit tests🫠

**Pull request merged**